api/src/test/java/org/motechproject/nms/api/osgi/KilkariControllerBundleIT.java

```diff
@@ -38,10 +39,8 @@
 
 import javax.inject.Inject;
 import java.io.IOException;
-import java.util.Arrays;
-import java.util.HashSet;
-import java.util.List;
-import java.util.Set;
+import java.util.*;
```

Comment:
Doesn't PMD complain about .* imports?  There is a setting in intelij to stop it from doing this.